docs: misc docs changes (#2416)
AztecProtocol/aztec-packages: docs/docs/dev_docs/contracts/syntax/functions.md

---
title: Functions

> description: This page will go over functions, how private and public functions differ and how then can be used together.

Reviewer: ```suggestion
description: This page covers functions, private and public functions composability, as well as their differences.
```
